packages/bff/src/services/tenantService.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable functional/immutable-data */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+
+import {
+  tenantApi,
+  attributeRegistryApi,
+  bffApi,
+} from "pagopa-interop-api-clients";
+
+export function bffServiceBuilder() {
+  return {};
+}
+export type BffService = ReturnType<typeof bffServiceBuilder>;
+
+export function enhanceTenantAttributes(
+  tenantAttributes: tenantApi.TenantAttribute[],
+  registryAttributes: attributeRegistryApi.Attribute[]
+): bffApi.TenantAttributes {
+  const registryAttributesMap: Map<string, bffApi.Attribute> = new Map(
+    registryAttributes.map((attribute) => [attribute.id, attribute])
+  );
+
+  const declared = tenantAttributes
+    .map((attr) => getDeclaredTenantAttribute(attr, registryAttributesMap))
+    .filter((x): x is bffApi.DeclaredTenantAttribute => x !== undefined);
```

Comment:
`x` too much generics
```suggestion
    .filter((att): att is bffApi.DeclaredTenantAttribute => att !== undefined);
```